src/pack/index.ts

```diff
@@ -20,10 +20,11 @@ export type PackProperties = {
   maxChunkSize?: number,
   maxChildrenPerNode?: number,
   wrapWithDirectory?: boolean,
-  hasher?: MultihashHasher
+  hasher?: MultihashHasher,
+  shardSplitThreshold?: number 
```

Comment:
linter failing here with:

```
src/pack/index.ts:24:31
ERROR: 24:31  no-trailing-whitespace    trailing whitespace
```

```suggestion
  shardSplitThreshold?: number
```